openapi.yml

```diff
@@ -443,6 +448,15 @@ components:
     Workflow:
       allOf:
         - {$ref: '#/components/schemas/Resource'}
+    WorkflowArgs:
+      description: The arguments needed to run the Workflow
+      type: object
+      required: ['workspace_id']
+      properties:
+        workspace_id: {type: string}
```

Comment:
This seems redundant, since the `workflow_id` is already in the request URL. We should either remove it from the endpoint (`POST /workspace/{workspace-id}` -> `POST /workspace`) or from the `WorkflowArgs, otherwise there's a risk of inconsistency.

Reply:
I think you are confusing "workflow" and "workspace", please correct me if I am wrong. The `workflow_id` is submitted via the url-path and the `workspace_id` is submitted in the json-body.
But maybe you are still right and it would be better to remove `workspace_id` and expect it in the `properties`-Object. But as I thought the workspace_id is required always, I set it as a separate parameter.